roadmap/implementers-guide/src/runtime/router.md

```diff
@@ -27,6 +38,52 @@ No initialization routine runs for this module.
 
 ## Routines
 
+There are two routines intended for use by the relay chain extrinsics: `ensure_downward_messages_fits`
```

Comment:
nit: `ensure_downward_messages_fit` sounds a bit nicer

Reply:
ah, sure, that's a typo actually. -s works only with 3rd p. singular. I got `ensure_horizontal_messages_fit` right tho : )